.github/workflows/split-docker-build.yml

```diff
@@ -0,0 +1,254 @@
+                        org.opencontainers.image.description=Resoto Shell (resh)
+                        org.opencontainers.image.vendor=Some Engineering Inc.
+
+            -   name: Set up QEMU
+                id: qemu
+                uses: docker/setup-qemu-action@v1
+                with:
+                    image: tonistiigi/binfmt:latest
+                    platforms: arm64,amd64
+
+            -   name: Set up Docker Buildx
+                id: buildx
+                uses: docker/setup-buildx-action@v1
+
+            -   name: Cache Docker layers
+                uses: actions/cache@v2
+                with:
+                    path: /tmp/.buildx-cache
+                    key: ${{ runner.os }}-buildx-${{ github.sha }}
+                    restore-keys: |
+                        ${{ runner.os }}-buildx-
+
+            -   name: Log in to Docker Hub
+                if: github.event_name != 'pull_request'
+                uses: docker/login-action@v1
+                with:
+                    username: ${{ secrets.DOCKERHUB_USER }}
+                    password: ${{ secrets.DOCKERHUB_PASS }}
+
+            -   name: Log in to GitHub Container Registry
+                if: github.event_name != 'pull_request'
+                uses: docker/login-action@v1
+                with:
+                    registry: ghcr.io
+                    username: ${{ github.actor }}
+                    password: ${{ secrets.GITHUB_TOKEN }}
+
+            -   name: Build and push resotobase Docker image
+                uses: docker/build-push-action@v2
+                with:
+                    context: .
+                    file: ./Dockerfile.resotobase
+                    platforms: ${{ steps.platform.outputs.targets }}
+                    push: ${{ github.event_name != 'pull_request' }}
+                    build-args: |
+                        SOURCE_COMMIT=${{ github.sha }}
+                        TESTS=false
+                    tags: ${{ steps.basemeta.outputs.tags }}
+                    labels: ${{ steps.basemeta.outputs.labels }}
+                    cache-from: type=local,src=/tmp/.buildx-cache
+                    cache-to: type=local,dest=/tmp/.buildx-cache-new
+
+            -   name: Build and push resotocore Docker image
+                if: github.event_name != 'pull_request'
+                uses: docker/build-push-action@v2
+                with:
+                    context: .
+                    file: ./Dockerfile.resotocore
+                    platforms: ${{ steps.platform.outputs.targets }}
+                    push: ${{ github.event_name != 'pull_request' }}
+                    build-args: |
+                        IMAGE_TAG=${{ steps.sha.outputs.short }}
+                    tags: ${{ steps.coremeta.outputs.tags }}
+                    labels: ${{ steps.coremeta.outputs.labels }}
+                    cache-from: type=local,src=/tmp/.buildx-cache
+                    cache-to: type=local,dest=/tmp/.buildx-cache-new
```

Comment:
This will overwrite the cache, and I think we only need to cache for the base image.